lib/mod/cache/mod.go

```diff
@@ -0,0 +1,14 @@
+package cache
+
+import "strings"
+
+func splitMod(mod string) (remote, owner, repo string) {
```

Comment:
Maybe call this `parseModURL` and change the filename to `util.go`

Reply:
wilco